### PR TITLE
Piros/orgpage404fix

### PR DESCRIPTION
**Pull request by pietrorosa77**
Fix for JIIRA issue

https://sourcegraph.atlassian.net/browse/CLOUD-137

### CODE REVIEW

File: client/web/src/org/settings/OrgSettingsArea.tsx [hunk 1]
```diff
@@ -31,6 +31,8 @@ const NotFoundPage: React.FunctionComponent = () => (
     />
 )
 
+const WaitingComponent: React.FunctionComponent = () => <LoadingSpinner className="m-2" />
```
Comment (kopancek) [inline client/web/src/org/settings/OrgSettingsArea.tsx:34]:
```suggestion
const LoadingComponent: React.FunctionComponent = () => <LoadingSpinner className="m-2" />
```

File: client/web/src/org/settings/OrgSettingsArea.tsx [hunk 2]
```diff
@@ -44,13 +46,16 @@ export const OrgSettingsArea: React.FunctionComponent<Props> = props => {
     // we can ignore the loading and error states in this case
     // if there is an error, we will not show the code host connections and repository screens
     // same for until the feature flag value is loaded (which in practice should be fast)
```
Comment (kopancek) [inline client/web/src/org/settings/OrgSettingsArea.tsx:48]: I think we should change the comment as we do not ignore the loading component anymore
```suggestion
    // we can ignore the error state in this case
    // if there is an error, we will not show the code host connections and repository screens
```

Comment (pietrorosa77) [reply client/web/src/org/settings/OrgSettingsArea.tsx]: I added your change locally

File: client/web/src/org/settings/OrgSettingsArea.tsx (hunk 1 above)
Comment (pietrorosa77) [reply client/web/src/org/settings/OrgSettingsArea.tsx]: Added your change locally

**Pull request closed without merging**